src/drt/src/io/io_guide.cpp

```diff
@@ -676,11 +676,14 @@ bool io::Parser::genGuides_gCell2APInstTermMap(
   frMTerm* trueTerm = instTerm->getTerm();
   std::string name;
   frInst* inst = instTerm->getInst();
-  dbTransform shiftXform = inst->getTransform();
-  shiftXform.setOrient(dbOrientType(dbOrientType::R0));
+  dbTransform shiftXform;
 
   int pinIdx = 0;
   int pinAccessIdx = (inst) ? inst->getPinAccessIdx() : -1;
+  if (inst != nullptr) {
```

Comment:
This doesn't hurt but if instTerm->getInst() returns null something is wrong with odb.

Reply:
Perhaps we could have a critical here? We don't have a lot of criticals in the code, but it seems like a good case.

Reply:
it would make sense to have it in dbITerm::getInst if Coverity will then use that inter-procedurally.

Reply:
Like this: https://github.com/The-OpenROAD-Project/OpenROAD/pull/5561/files#diff-1bfa0367ed9744f752dcda88d6c1557285e18e9351bba5fcf6d4d67d4572129fR244-R247?

Reply:
yes